prebid-server/pbs-modules/confiant-ad-quality.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: page_v2
+page_type: pbs-module
+title: Prebid Server Confiant Ad Quality Module
+display_name : Confiant Ad Quality Module
+sidebarType : 5
+---
+
+# Confiant Ad Quality Module
+{:.no_toc}
+
+* TOC
+  {:toc }
```

Comment:
```suggestion
{:toc }
```

Indentation must be removed otherwise it's not rendered properly: 
https://deploy-preview-4815--prebid-docs-preview.netlify.app/prebid-server/pbs-modules/confiant-ad-quality.html